backend/python/app/tests/graphql/mutations/test_story_mutation.py

```diff
@@ -1,7 +1,7 @@
 from datetime import datetime
```

Comment:
do you have to use StoryAll and StoryContentAll to pass these tests?

Reply:
For this yes. When I revert the changes, `test_create_story` fails